Medable/ResearchStack
# PATAND-20: Remove unused strings from Research Stack project.

### Description 
This MR consists in removing old strings that are no longer in used for RS or any other project.

The following entries were removed: 

```<string name="rsb_consent_learn_more">Learn More</string> 
<string name="rsb_consent_share_title">Sharing Options</string> 
<string name="rsb_consent_share_widely">Share my data with %1$s and qualified researchers 
    worldwide
</string>
<string name="rsb_consent_share_no">Not sharing your data</string> 
<string name="rsb_consent_share_description"><![CDATA[%1$s will receive your study data from your
    participation in this study. \n\nSharing your coded study data more broadly (without
    information such as your name) may benefit this and future research.<br/><br/><a href="%2$s">
    Learn more about data sharing</a>]]>
</string> 
<string name="rsb_consent_share_only">Only share my data with %1$s</string> 
<string name="rsb_consent_review_title">Review</string> 
<string name="rsb_consent_review_reason">By agreeing you confirm that you read the information
    and that you wish to take part in this research study.
</string>  
<string name="rsb_consent_review_instruction">Review the form below, and tap Agree if you\'re
    ready to continue.
</string> 
<string name="rsb_consent_form_title">Consent</string> 
<string name="rsb_consent_name_full">Full Name</string> 
<string name="rsb_consent_name_placeholder">Required</string> 
<string name="rsb_consent_signature_content">By agreeing you confirm that you read the consent
    and that you wish to take part in this research study.
</string> 


<string name="rsb_quiz">Quiz</string> 
<string name="rsb_quiz_evaluation">Quiz Evaluation</string> 
<string name="sb_quiz_evaluation_try_again">Try Again</string> 
<string name="rsb_quiz_evaluation_review_consent">Review Consent</string> 
<string name="rsb_quiz_evaluation_retake">Retake Quiz</string> 
<string name="rsb_quiz_evaluation_great_job">Great Job!</string> 
<string name="rsb_submit">Submit</string> 
<string name="rsb_exit">Exit</string> 
<string name="rsb_get_started">Get Started</string> 
<string name="rsb_what_to_expect">What to Expect</string> 
<string name="rsb_name">Name</string> 
<string name="rsb_email">Email</string> 
<string name="rsb_ok">OK</string> 
<string name="rsb_yes">Yes</string> 
<string name="rsb_no">No</string> 
<string name="rsb_error_data_failed">Failed to load encrypted data, try again!</string> 
<string name="rsb_pincode_enter_title">Enter your passcode</string> 
<string name="rsb_pincode_enter_summary">Enter your %1$d-%2$s code to confirm your identity.</string> 
<string name="rsb_pincode_enter_digit">digit</string> 
<string name="rsb_pincode_enter_letter">letter</string> 
<string name="rsb_pincode_enter_character">character</string> 
<string name="rsb_pincode_enter_error">That code doesn’t match the one you entered, Try again!</string> 

<string name="rsb_error_assetvideoview_title">Video problem</string> 
<string name="rsb_consent_doc_line_date_format">"MM-dd-yyyy"</string> 
````


### How to test
* Checkout task/PATAND-20-Remove_old_strings in PAT, AXON and RS
* Run ./gradlew clean assembleDebug test

EXPECTED OUTPUT: 
* You should have a successful build
